server/database/connection.js

```diff
@@ -16,6 +16,8 @@ const connectDatabase = async () => {
 };
 module.exports = connectDatabase;
 
+
+
```

Comment:
Avoid random changes like this cause it can just cause merge conflicts for no reason